XamlControlsGallery/App.xaml

```diff
@@ -50,24 +56,41 @@
             <!--  Breakpoints  -->
             <x:Double x:Key="Breakpoint640Plus">641</x:Double>
 
-            <x:String x:Key="AppName">All controls</x:String>
+            <x:String x:Key="ControlsName">All controls</x:String>
+            <x:String x:Key="NewControlsName">What's New</x:String>
 
             <Style x:Key="OutputTextBlockStyle" BasedOn="{StaticResource BodyTextBlockStyle}" TargetType="TextBlock">
                 <Setter Property="Margin" Value="8,8,0,0" />
             </Style>
-
+           
+            <Style x:Name="NarrowHeaderTextBlockStyle" BasedOn="{StaticResource HeaderTextBlockStyle}" TargetType="TextBlock">
+                <Setter Property="Foreground" Value="{StaticResource SystemControlForegroundChromeWhiteBrush}"/>
+                <Setter Property="VerticalAlignment" Value="Top"/>
+                <Setter Property="HorizontalAlignment" Value="Left"/>
+                <Setter Property="Margin" Value="36,12,0,0"/>
+            </Style>
```

Comment:
Can we align with the caption button heights? My guess is "probably not" based on the height of the hamburger and back buttons, but this looks so strange to me: 

![image](https://user-images.githubusercontent.com/25991996/65172965-c451dd80-da02-11e9-8402-cb7d0e8a28a8.png)

Reply:
Unfortunately no, because the caption controls are 32px tall and the back/hamburger buttons are 40px tall. But I do want to take another pass over Minimal mode in general because there are several (most) things I don't like about it. I mostly want to get this massive change in first so I can take some time to play with just the narrow app state of things.